packages/forms-web-app/src/lib/dashboard-functions.js

```diff
@@ -0,0 +1,31 @@
+const extractAppealNumber = (caseReference) => {
+	return caseReference.split('/').pop();
```

Comment:
We won't need this going forward - the caseReference will be just a number.

Reply:
I will leave this in for now but will refactor it away in future - it should still work for now with a number